.drone.yml

```diff
@@ -191,51 +192,26 @@ volumes:
     temp: {}
 
 services:
-  - name: mysql
-    image: mysql:5.7
-    pull: always
-    environment:
-      MYSQL_ALLOW_EMPTY_PASSWORD: yes
-      MYSQL_DATABASE: test
-
-  - name: mysql8
-    image: mysql:8
-    pull: always
-    environment:
-      MYSQL_ALLOW_EMPTY_PASSWORD: yes
-      MYSQL_DATABASE: testgitea
-
-  - name: mssql
-    image: mcr.microsoft.com/mssql/server:latest
-    pull: always
+  - name: pgsql
+    pull: default
+    image: postgres:10
```

Comment:
`10`?
Isn't that quite outdated now?

Reply:
Good shout, just a copy/paste. I'll look at a newer version.

Reply:
```suggestion
    image: postgres:15
```